common/selectors/selectAndDedupe.js

```diff
@@ -44,3 +45,9 @@ module.exports = function selectAndDedupe(group) {
     return result;
   }, []);
 };
+
+module.exports.dedupeOne = list => {
+  dedupe.defaults.createKey = site => (site ? site.hostname : Math.random());
```

Comment:
so, basically, this function dedupes based on domain name? Maybe add a doc string or comment or something.